### PR TITLE
Release 20210317

### PR DESCRIPTION
**Pull request by ElenaFdR**
## Review checklist template

- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] Correct Markdown and formatting
- [x] All content reviewed

-----
[View rendered activities/hr-management/contracts.md](https://github.com/publiccodenet/about/blob/20210317/activities/hr-management/contracts.md)
[View rendered activities/hr-management/index.md](https://github.com/publiccodenet/about/blob/20210317/activities/hr-management/index.md)
[View rendered activities/hr-management/onboarding.md](https://github.com/publiccodenet/about/blob/20210317/activities/hr-management/onboarding.md)
[View rendered activities/hr-management/probation.md](https://github.com/publiccodenet/about/blob/20210317/activities/hr-management/probation.md)
[View rendered activities/hr-management/salary-administration.md](https://github.com/publiccodenet/about/blob/20210317/activities/hr-management/salary-administration.md)
[View rendered activities/staff-management/expense.md](https://github.com/publiccodenet/about/blob/20210317/activities/staff-management/expense.md)
[View rendered activities/staff-management/sickness.md](https://github.com/publiccodenet/about/blob/20210317/activities/staff-management/sickness.md)

### CODE REVIEW

**Comment by Ainali** [comment]
Hmm, should hr-management/index.md be linked from the main activities index perhaps?

**Comment by bvhme** [comment]
this is great content and I think it is good to go.

Can I propose one change, changing HR management to 'employment management' or 'staff contact management' or something similar. 

I'm personally quite a fervent opposer of the term HR as I feel it dehumanizes people into being resources. Apart from it being an acronym of course :)

Let me know if that is a reasonable request.

**Comment by ElenaFdR** [comment]
@bvhme Avoiding "HR" is totally reasonable; I'm also not a fan. I suggested we change 'how to staff' on chat.publiccode.net in Dec 2020:

> @demkodo is planning to put her operations handbook on About. A lot of it is how to operate the procedures set out in staff management. As an example, the corollary to 'how to report illness' is 'how to manage an illness report'. To make it easier to find the things you need as staff, rather than @demkodo as operations coordinator, I propose separate folders for the activities. 'Staff management' feels like a better fit for the stuff @demkodo does; is there a better description than 'being managed' that you would prefer?
> (This stuff is usually called 'HR policies', but our About structure encourages us to structure this by activities and keep it active.)
> (We also have 'how to staff', which could be expanded from staff onboarding to staff onboarding and policies.)

File: activities/finance-management/bank.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Our bank is the Rabobank. There is a EUR checking and savings account. We are in
 
 These people have bank access:
 
-* The Chief Executive, Operations Coordinator, and Treasurer of the board have an online key for internet banking
-* The Chief Executive has a debit card (wereldpas) and credit card
-* The Operations Coordinator is authorized to make € 5000 payments without a co-sign
-* The Chief Executive’s daily limit is € 50,000 and can be temporarily raised if needed by calling the bank
+* The chief executive, operations coordinator, and treasurer of the board of directors have an online key for internet banking
```
Comment (bvhme) [inline activities/finance-management/bank.md:12]: Hey, this is incorrect: it is the treasurer (of the vereniging).

```suggestion
* The chief executive, operations coordinator, and treasurer have an online key for internet banking
```

**Review by bvhme — APPROVED** [review]
This is a great improvement :) Can go in like this

**Review by bvhme — DISMISSED** [review]

**Comment by bvhme** [comment]
This build is most definetely not working. Travis is right and GitHub actions seems to not even run the tests at all.

**Comment by ElenaFdR** [comment]
I've now fixed the broken internal links flagged by Travis. Instead, it's failing on links that work in the browser (per usual!) I'm not sure what's up with Github Actions - @ericherman or @felixfaassen, do you have a moment to work through this with me sometime this week?

**Review by nebster — APPROVED** [review]
Looks good!

**Pull request merged**